### PR TITLE
Adding abutters app functionality

### PR DESCRIPTION
**Pull request by paylakatel**
This PR:
- removes all unnecessary vision zero artifacts and functionality 
- adds in all abutters app functionality

At a high level we are:
1. Letting people search for an address or click on the map to select a parcel.
2. Using the location of that click to query our hosted parcel layer (hosted on [BostonMaps](http://boston.maps.arcgis.com/home/index.html) by the Analytics Team) and figure out which parcel the user has clicked on/inside. We are currently using placeholder data - the app will be updated when the final data is available.
3. Using an ESRI buffer api (hosted on a City AWS account) to buffer the selected parcel and display the buffered geometry on the map.
4. Using that buffer shape the query the parcel layer again and find all parcels that intersect with it (e.g. finding all parcels within x feet of the selected one).
5. We gather the ownership information from the intersecting parcels and put it into a csv users can download.

### CODE REVIEW

File: components/Layout.js [hunk 1]
```diff
@@ -51,53 +51,129 @@ export default class Layout extends React.Component {
             font-family: 'Lora';
             font-size: 18px;
           }
-
+          p {
```
Comment (phillipbentonkelly) [inline components/Layout.js]: Maybe you can put all this css in its own file and [import](https://codeburst.io/4-four-ways-to-style-react-components-ac6f323da822) it into this layout.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,4 @@
-# Vision Zero Crash Map
-
-This repo houses code for Boston's [vision zero crash map](https://apps.boston.gov/vision-zero/). The data displayed
-on the map can be found on the City's open data portal, under [vision zero crash records](https://data.boston.gov/dataset/vision-zero-crash-records)
-and [vision zero fatality records](https://data.boston.gov/dataset/vision-zero-fatality-records).
+# Abutters App
```
Comment (phillipbentonkelly) [inline README.md]: I think we can still use some of the readme from vision-zero and just rewrite it, specially the link to staging and prod. Also, some of the description on this PR can be useful.

File: components/Layout.js (hunk 1 above)
Comment (paylakatel) [reply components/Layout.js]: Just moved this to an external file - lmk what you think!

File: README.md (hunk 2 above)
Comment (paylakatel) [reply README.md]: 👍added more info to the readme

**Pull request merged**